diff --git a/xmake.lua b/xmake.lua
index cdbb05bbb..9918e0123 100644
--- a/xmake.lua
+++ b/xmake.lua
@@ -91,16 +91,16 @@ if is_plat("linux") and (linuxos.name() == "debian" or linuxos.name() == "ubuntu
     end
 else
 -- Let xrepo manage the dependencies for macOS and other GNU/Linux distros
-    add_requires("libpng 1.6.37", {system=false})
-    add_requires("libiconv 1.17", {system=false})
-    add_requires("zlib 1.2.12", {system=false})
-    add_requires("libjpeg v9e", {system=false})
-    add_requires("libcurl 7.84.0", {system=false})
-    add_requires("freetype 2.12.1", {system=false})
+    add_requires("libpng >=1.6.37", {system=true})
+    -- add_requires("libiconv >=1.17", {system=true})
+    add_requires("zlib >=1.2.12", {system=true})
+    add_requires("libjpeg", {system=true})
+    add_requires("libcurl >=7.84.0", {system=true})
+    add_requires("freetype >=2.12.1", {system=true})
 end
 
 if is_plat("mingw") then
-    add_requires("nowide_standalone 11.2.0", {system=false})
+    add_requires("nowide_standalone 11.2.0", {system=true})
 end
 
 if is_plat("linux") then
@@ -108,7 +108,7 @@ if is_plat("linux") then
 end
 
 local PDFHUMMUS_VERSION = "4.1"
-add_requires("pdfhummus "..PDFHUMMUS_VERSION, {system=false,configs={libpng=true,libjpeg=true}})
+-- add_requires("pdfhummus >="..PDFHUMMUS_VERSION, {system=false,configs={libpng=true,libjpeg=true}})
 add_requires("s7 2023.04.13", {system=false})
 
 
